api/utils.py

```diff
@@ -0,0 +1,60 @@
+import os
+import io
+import pandas as pd
+from api.models import Patient, Visit, Vitals, Consult, Diagnosis, Medication, Order, CustomUser, MedicationReview
+
+def extract_data_into_obj():
+        # Define the models and their corresponding sheet names
+        models = {
+            'patients': Patient,
+            'clinics': Visit,
+            'medicines': Vitals,
+            'appointments': Consult,
+            'diagnoses': Diagnosis,
+            'medication': Medication, 
+            'order': Order,
+            'customUser': CustomUser, 
+            'medicationReview': MedicationReview
+        }
+
+        # Directory where CSV files will be temporarily saved
+        temp_csv_dir = 'temp_csv'
+        os.makedirs(temp_csv_dir, exist_ok=True)
+
+        # Save each model's data to a separate CSV file
+        for table_name, model in models.items():
+            queryset = model.objects.all().values()
+            df = pd.DataFrame(list(queryset))
+            csv_filename = os.path.join(temp_csv_dir, f"{table_name}.csv")
+            df.to_csv(csv_filename, index=False)
+            # self.stdout.write(self.style.SUCCESS(f'Data from table {table_name} exported to {csv_filename}.'))
```

Comment:
Small nitpick: Try not to leave commented out code in the commit.

Reply:
apologies, noted